### PR TITLE
Add Bucket icon

### PR DESCRIPTION
**Pull request by undeletable**
Add Bucket icon as requested in https://github.com/grommet/grommet-icons/issues/268

### CODE REVIEW

File: src/js/__tests__/__snapshots__/metadata-test.js.snap [hunk 1]
```diff
@@ -338,6 +338,11 @@ Object {
     "tools",
     "paint",
   ],
+  "Bucket": Array [
+    "cloud",
```
Comment (jcfilben) [inline src/js/__tests__/__snapshots__/metadata-test.js.snap:342]: Hmm I'm not sure if I would consider cloud as part of the metadata for bucket.

**Review by jcfilben — COMMENTED** [review]
Thanks for working on this! Can you also export the bucket icon in the icons/index.d.ts file?

**Comment by undeletable** [comment]
@jcfilben fixed that

**Review by jcfilben — APPROVED** [review]
Looks good!

**Pull request merged**